MAUI/Cartesian-Charts/Zooming-and-panning.md

```diff
@@ -248,4 +248,132 @@ chart.ZoomPanBehavior = zooming;
 
 {% endhighlight %}
 
+{% endtabs %}
+
+## Selection zooming
+
+Selection zooming feature allows users to interactively choose a particular area of the chart and zoom in. By specifying [EnableSelectionZooming]() property to `true` as shown in the following code snippet, you can double tap and drag to select a range on the chart to be zoomed in. The default value of this property is false.
+
+{% tabs %}
+
+{% highlight xml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.ZoomPanBehavior>
+        <chart:ChartZoomPanBehavior EnableSelectionZooming ="True"/>
+    </chart:SfCartesianChart.ZoomPanBehavior>
+    ...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+ChartZoomPanBehavior zooming = new ChartZoomPanBehavior()
+{
+    EnableSelectionZooming = true
+};
+
+chart.ZoomPanBehavior = zooming;
+...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Selection zooming support in MAUI Chart](Zooming-and-panning_images/maui_selection_zooming.gif)
+
+### Selection rectangle customization
+
+You can customize the selection rectangle using the below properties.
+
+* [SelectionRectStrokeWidth]() - used to change the stroke width of selection rectangle
+
+* [SelectionRectStroke]() - used to change the stroke color of selection rectangle.
+
+* [SelectionRectStrokeDashArray]() - used to change the stroke dashes of selection rectangle.
+
+* [SelectionRectFill]() - used to change the fill color of selection rectangle.
+
+### Show trackball axis label
+The selection zooming trackball axis label is enabled by setting the [ShowTrackballLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ShowTrackballLabel) property to `true`. The default value of the [ShowTrackballLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ShowTrackballLabel) is `false`. The trackball axis labels can be customized using following properties:
+
```

Comment:
The trackball axis labels can be customized using following properties:
 
 From where user can find this properties?
 
 we not provided support for label alignment. if the support not given remove the property from this list.

Reply:
mentioned property and removed label alignment property